.github/workflows/test_extensions.yml

```diff
@@ -60,12 +60,6 @@ jobs:
         case ${{ matrix.extension }} in
           datalad-metalad)
             DL_PIP_INSTALLS+=" nose";;
-          datalad-neuroimaging)
-            DL_TESTER=nose;;
-          datalad-hirni)
-            DL_REPO=psychoinformatics-de/${{ matrix.extension }}
-            DL_TESTER=nose
-            DL_NEED_SINGULARITY=1;;
```

Comment:
note: hirni is no longer tested and neuroimaging pytest-ified

Reply:
Yes, that's why I wanted to remove it, since this file kinda suggests a different reality.